### PR TITLE
Documentation generation

### PR DESCRIPTION
**Pull request by NachoSoto**
Fixes [CSDK-325]

[CSDK-325]: https://revenuecats.atlassian.net/browse/CSDK-325?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -130,32 +154,53 @@ jobs:
 
   runtest:
     docker:
-      - image: circleci/node:lts
+      - image: cimg/ruby:2.6
```
Comment (NachoSoto) [inline .circleci/config.yml:157]: This has `bundler`, and `node` is installed through `install-nvm`.

**Comment by NachoSoto** [comment]
This is ready 🎉

File: .circleci/config.yml [hunk 2]
```diff
@@ -85,6 +109,12 @@ commands:
       - run: echo 'nvm use 16.13.1' >> $BASH_ENV
       - run: node -v
 
+  set_up_git_user:
+    description: Set git config parameters
+    steps:
+      - run: git config --global user.email "dev+ci@revenuecat.com"
+      - run: git config --global user.name "CircleCI"
```
Comment (aboedo) [inline .circleci/config.yml:116]: 👌 
We could even call it `RevenueCat CI`

Comment (aboedo) [reply .circleci/config.yml:116]: we can also set these as env variables, but it might be overkill for now

File: RELEASING.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 1. Create a release branch- `release/x.y.z`
 1. Update versions in `VERSIONS.md`
 1. Update to the latest purchases-hybrid-common version in `plugin.xml`
-1. Update version in `package.json`, `plugin.xml`, `PurchasesPlugin.java`, and `PurchasesPlugin.swift`.
+1. Update version in `package.json`, `plugin.xml`, `PurchasesPlugin.java`, `PurchasesPlugin.swift`, and `scripts/docs/index.html`.
```
Comment (aboedo) [inline RELEASING.md:4]: looking forward to this getting automated away through the fastlane plugin

**Review by aboedo — APPROVED** [review]
:chefskiss:

File: .circleci/config.yml (hunk 2 above)
Comment (NachoSoto) [reply .circleci/config.yml:116]: I thought about it, but then we might have to validate that they're defined so we don't end up pushing with no user / email again?
I'll change it to `RevenueCat CI` 👍🏻

**Comment by NachoSoto** [comment]
@aboedo _insert Spiderman meme_

**Pull request merged**